framework/graphics/fps_info.cpp

```diff
@@ -208,19 +210,21 @@ void FpsInfo::LogToConsole()
         {
             GFXRECON_WRITE_CONSOLE("Load time:  %f seconds", GetElapsedSeconds(start_time_, replay_start_time_));
         }
-        GFXRECON_WRITE_CONSOLE("Total time: %f seconds", GetElapsedSeconds(start_time_, measurement_end_time_));
+        GFXRECON_WRITE_CONSOLE("Total time: %f seconds",
+                               GetElapsedSeconds(start_time_, static_cast<uint64_t>(measurement_end_time_)));
 
         WriteFpsToConsole("Replay FPS:",
-                          replay_start_frame_,
-                          measurement_end_frame_ - 1 + replay_start_frame_ - 1,
-                          replay_start_time_,
+                          static_cast<uint64_t>(replay_start_frame_),
```

Comment:
yes, chances are we should change the type of some members, rather that casting back/forth in other places.
I wanted to avoid to get too creative with a rewrite here, it's not the focus.
the result is this rather mechanical "fixing of warnings" refactoring, 
it should just make visible what the implication from higher warning-level result to

Reply:
@fabian-lunarg @jzulauf-lunarg can this be a separate PR?  Make a one-liner issue to make sure it doesn't get forgotten?

Reply:
#1907